src/test/test_map.c

```diff
@@ -271,22 +389,6 @@ static void test_map_delete_unknown(void **state)
         assert_int_equal(NOT_FOUND_ERROR, map_delete(map, "unknown"));
 }
 
-static void test_map_delete_resize(void **state)
```

Comment:
This was a busted test before and needs to be refactored when we add resizing the map downwards